### DISCUSSION TITLE
YouTube | Patch | Downloads

### DISCUSSION THREAD

**Discussion by TurboWrestler250** [ReVanced/revanced-discussions#1065] [Question] [answered]
How do I download videos from YouTube patched?

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
Install a third party download app such as [NewPipe](https://newpipe.net),

then enable one or both of the download buttons in ReVanced settings under `Settings -> Player -> External downloads`


But if you want to use the native in app offline watch feature then the only way is to pay for YT Premium.

**Reply by bazixv13** [reply]
I personally find Seal better it has more use cases it can also download tiktoks Instagram reels and more
